ga4gh/frontend.py

```diff
@@ -496,6 +496,18 @@ def searchVariants():
         flask.request, app.backend.runSearchVariants)
 
 
+@DisplayedRoute('/variantannotationsets/search', postMethod=True)
+def searchVariantAnnotationSets():
```

Comment:
Variant annotations and sets can currently be found by search, but we need to be able to find them by get requests as well.